shared-libs/cht-datasource/.eslintrc.js

```diff
@@ -64,6 +64,12 @@ module.exports = {
         }],
         ['jsdoc/check-tag-names']: ['error', { definedTags: ['typeParam']}],
       }
+    },
+    {
+      files: ['*.spec.ts'],
+      rules: {
+        ['@typescript-eslint/no-unused-expressions']: 'off',
```

Comment:
The vast majority of the new linting errors were due to the calls in the test like:

```ts
expect(isContactTypeQualifier.notCalled).to.be.true;
```

I could not find any eslint/chai-native approach that would resolve this issue, so seems file to just disable this rule for the tests...